libqtile/ipc.py

```diff
@@ -57,7 +67,11 @@ def _unpack(self, data):
     def _unpack_body(self, body):
         return marshal.loads(body)
 
-    def _pack(self, msg):
+    def _pack(self, msg, is_json=False):
```

Comment:
I feel like you should have a separate `_pack_json()` method, since you know if you need to pack json.